_docs/principles/accessibility.md

```diff
@@ -3,29 +3,67 @@ title: Accessibility
 category: Principles
 usage: >
-  Visual accessibility standards enable users with varied levels of color impairment to successfully navigate and use a Helix-based UI.
+  Accessibility standards promote the design of user interfaces (UI) so they can be used by people with the widest range of abilities within the widest range of situations.
 status: stable
 resource: true
-last-modified: 2017-10-24
+last-modified: 2019-04-25
 ---
 
 {% include toc.html %}
 
 <section class="static-section" markdown="1">
 
-## Accessibility for visual design
-
 <div class="hxRow"  markdown="1">
 
-{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-12-md hxSpan-12-lg" %}
+{% column left:"hxCol hxSpan-12-xs hxSpan-12-sm hxSpan-8-md hxSpan-8-lg" %}
+
+## Understanding accessibility
+
+An accessible UI is designed so that it can be *accessed* and used by people who suffer from a permanent or a temporary impairment. While visual impairments such as color blindness and lack of sight are two of the most obvious impairments to consider, an impairment can also be auditory, physical, or cognitive. For example, a user who suffers from a concussion may experience difficulty processing information, so ensure that you write succinct on-page instructions and labels. A user who experiences hand tremors as a side effect to medication has difficulty operating a mouse, so design pointer targets so they are large enough.
+
+Designing for accessibility is best practice because you consider all users and not just those with a permanent disability.
 
-Rackspace products must be made accessible for everyone&mdash;regardless of a person’s abilities. We can assume from the statistics that many of our users have some form of visual impairment. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
+The purpose of this document is to articulate guidelines that you can use when you design control panel page. The list of guidelines is not exhaustive. Refer to [Web Content Accessibility Guidelines 2.1 (WCAG)](https://www.w3.org/TR/WCAG21/) for a complete list of guidelines.
+
+### Why accessibility is important
+
+Accessibility is more than just good design. It's about the empathy for and inclusion of all people. Consider sight. [Colour Blind Awareness](http://www.colourblindawareness.org/) estimates that:
 
 - 4.5% of the global population experience color blindness (1 in 12 men and 1 in 200 women)
 - 4% suffer from low vision (1 in 30 people)
 - 0.6% are blind (1 in 188 people)
 
-By using the following best practices, you can help ensure that your design is accessible for the visually impaired.
+With a global population of just over 7.5 billion people, 9.1% of the world's population equates to over one-half billion visually-impaired user that cannot use an inaccessible product. After factoring in the myriad other forms of permanent and temporary impairments, designing inaccessible products can have a material and negative impact on a large percent of users.
```

Comment:
simplify the "after myriad" sentence. the first half seems unnecessary and the second half is confusing language.

Reply:
Done.